### ISSUE TITLE
[COGNITION]: Alert Boxes - Icons SHOULD be hidden from screen readers

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#10463]
# [508-defect-3](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-3)

Our [Alert Box component](https://department-of-veterans-affairs.github.io/veteran-facing-services-tools/visual-design/components/alertbox/) uses icons on the left, that are appearing as empty tab stops for JAWS. These do not announce to users and could be confusing. Let's hide them using the updated snippets below.

**Feedback framework**

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Point of Contact

**VFS Point of Contact:** _Trevor_

## Acceptance Criteria

- [ ] Icons are not being announced to screen readers. JAWS in particular.

## Environment

* https://department-of-veterans-affairs.github.io/veteran-facing-services-tools/visual-design/components/alertbox/

## WCAG or Vendor Guidance (optional)

* [Name, Role, Value: Understanding SC 4.1.2](https://www.w3.org/TR/UNDERSTANDING-WCAG20/ensure-compat-rsv.html)

## Screenshots or Trace Logs (optional)
![Screen Shot 2020-06-18 at 5 15 42 PM](https://user-images.githubusercontent.com/934879/85077285-88ad9500-b187-11ea-8969-0244c7d04a5b.png)

**Comment by 1Copenut** [comment]
Issue moved to [department-of-veterans-affairs/vets-design-system-documentation #215](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/215) via [**ZenHub**](https://www.zenhub.com/)
